src-input/duk_hobject.h

```diff
@@ -36,6 +36,11 @@
  * inspection code.
  */
 
+/* FIXME: ES spec: all construct-only functions have [[Call]] and [[Construct]]
```

Comment:
Alright, I see.  I was just trying to figure out what needed to be "fixed" (since it's a FIXME).  As long as you know the answer to that, I'm satisfied. :)